rotkehlchen/chain/ethereum/uniswap/uniswap.py

```diff
@@ -220,6 +244,224 @@ def _get_known_asset_price(
 
         return asset_price
 
+    def _get_trades(
+            self,
+            addresses: List[ChecksumEthAddress],
+            from_timestamp: Timestamp,
+            to_timestamp: Timestamp,
+            db_query_get_used_query_range: Callable,
```

Comment:
Why are you passing 4 different methods of `DBHandler` as arguments to this function? Simply pass the db when initializing the uniswap object and access it with `self.database`

Reply:
As we agreed to avoid as much as possible hard mocking, I used fake injections, which I thought they are a valid and accepted strategy (unit tests only need functions). I will change it.